clang/runtime/dpct-rt/include/sparse_utils.hpp

```diff
@@ -598,6 +691,17 @@ class sparse_matrix_desc {
   matrix_format _data_format;
   std::optional<oneapi::mkl::uplo> _uplo;
   std::optional<oneapi::mkl::diag> _diag;
+  void *_temp_row_ptr = nullptr;
```

Comment:
Please add comment to tell the relation ship between _temp_row_ptr and _row_ptr.

Reply:
done